py/py.mk

```diff
@@ -157,7 +157,7 @@ PY_EXTMOD_O_BASENAME = \
 	extmod/moductypes.o \
 	extmod/modujson.o \
 	extmod/modure.o \
-	extmod/moduzlib.o \
+	extmod/moddeflate.o \
```

Comment:
If `extmod/moduzlib.c` it is not used anymore, can't we delete the file from the repo?

Reply:
The uzlib build is disabled in mpconfigport. I believe we won't need it anymore, but it is also not necessary to remove it now. My goal for the longer term is to move to recent, drastically changed, MicroPython releases.